lib/DimmableLightDevice.js

```diff
@@ -81,6 +90,7 @@ module.exports = {
 						this.parsedcurrentDimvalue = Math.round((currentDimValue / 254) * 100) / 100;
 						this.log('currentdimvalue: ', this.parsedcurrentDimvalue);
 						this.setCapabilityValue('dim', this.parsedcurrentDimvalue);
+						if (this.parsedcurrentDimvalue === 0) this.setCapabilityValue('onoff', false);
```

Comment:
moved check for `parsedcurrentDimvalue` to here so it does not have to be global and will also set the capability to off if the dim value is 0 before the transition time is over (which for instance could happen when dimming from 0.01 to 0 over a time of 30 minutes)